### PR TITLE
Topbar localization

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Removed `width` from `TopBarLink` - now it is useless
 * Fixed width of top bar links
 * Added russian translation

### CODE REVIEW

**Review by Cheshiriks — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/locales/Translations.kt [hunk 1]
```diff
@@ -49,3 +49,38 @@ const val TABLE_HEADERS_EN = """
...
+    "About us": "О нас",
+    "Propose vulnerability": "Предложите уязвимость",
+    "Vulnerabilities": "Уязвимости",
+    "Top Rating": "Рейтинг",
+    "SAVE Projects list": "SAVE-проекты",
+    "Vulnerabilities list": "Список уязвимостей"
+}
+"""
+
+const val TOPBAR_EN = """
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/locales/Translations.kt:70]: do we need it? why fallback is not working here?

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/locales/Translations.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/locales/Translations.kt:70]: It works, but if we want our Chinese colleagues help with translation, we should provide them a good-looking example of language they can understand

Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/locales/Translations.kt:70]: `English words` is a key already. I don't think that additional mapping `A` to `A` will help them, but it adds work for you

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/locales/Translations.kt:70]: Sometimes keys do not match with values

I'd like to leave as is for now

**Review by nulls — APPROVED** [review]

**Pull request merged**